tests/test_content.py

```diff
@@ -421,6 +421,83 @@ def test_datetime():
     assert new_result == '2001-02-03 04:05'  # output
 
 
+def test_nested_align():
+    """
+    # Nested Formats
```

Comment:
Hi :) Thank you very much for the patch! Could you just please replace this heading with "Nested Formats" as all other headings so far are using sentence-formatting? Thanks!